src/Config.js

```diff
@@ -37,11 +37,10 @@ export class Config {
 
     // Create a new DatabaseController per request
     if (cacheInfo.databaseController) {
-      const schemaCache = new SchemaCache(cacheInfo.cacheController, cacheInfo.schemaCacheTTL);
-      this.database = new DatabaseController(cacheInfo.databaseController.adapter, schemaCache);
+      this.database = new DatabaseController(cacheInfo.databaseController.adapter, cacheInfo.schemaCache);
```

Comment:
we should probably keep the cache / request too to prevent having caches that live too long across requests, particularly important in distributed environments without the `RedisCacheAdapter`. I'd love to hear your thoughts.